README.md

```diff
@@ -1,53 +1,116 @@
+🌐 Works across repositories (with a [PAT]) \
+📚 Pretty interface for Markdown docs \
+⤴️ Lets you open PRs for wiki docs
 
-## Before using this action
+## Usage
+
+![GitHub Actions](https://img.shields.io/static/v1?style=for-the-badge&message=GitHub+Actions&color=2088FF&logo=GitHub+Actions&logoColor=FFFFFF&label=)
+![GitHub wiki](https://img.shields.io/static/v1?style=for-the-badge&message=GitHub+wiki&color=181717&logo=GitHub&logoColor=FFFFFF&label=)
 
-You probably want to commit your wiki in its initial state before using this action.
-This is because this action will commit *all* differences between the wiki and the
-`.github/wiki` (or other specified directory) contents.
+Getting started is easy! Just create a GitHub Actions workflow file that uses
+spenserblack/actions-wiki and you're good to go! 🚀 Here's a ready-made example
+to help you blast off:
 
-```bash
-git clone https://github.com/octocat/my-repo.wiki.git .github/wiki
-rm -rf .github/wiki/.git
-git add .github/wiki
-git commit -m "Add wiki to repo"
+```yml
+name: Publish wiki
+on:
+  push:
+    branches: [main]
+    paths: [wiki/**, .github/workflows/publish-wiki.yml]
+concurrency:
+  group: wiki
+  cancel-in-progress: true
+permissions:
+  contents: write
+jobs:
+  wiki:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: spenserblack/actions-wiki@v0.1.1
+        with:
+          # Whatever directory you choose will be mirrored to the GitHub
+          # .wiki.git. The default is .github/wiki.
+          wiki-directory: wiki
+          # For now, you'll need to manually specify a GitHub token until we
+          # solve #2. The x: prefix is a dummy username.
+          token: x:${{ secrets.GITHUB_TOKEN }}
 ```
 
-## Usage
+<img align="right" alt="Screenshot of 'Create the first page' button" src="https://i.imgur.com/ABKIS4h.png" />
+
+❗ Make sure you turn on the GitHub wiki feature in your repo's settings menu.
+You'll also need to _manually_ create a dummy page to initialize the git repo
+that powers the GitHub wiki. If you don't, when we push to `your-repo.wiki.git`,
+your workflow will fail because the wiki doesn't exist.
+
+⚠️ You'll need to remember to enable the `contents: write` permission! GitHub
+recently [changed the default permissions granted to jobs] for new repositories.
+
+### Publishing to a different repository
+
+If you're pushing to a wiki that's **not the current repository** you'll need to
+get a [GitHub PAT] to push to it. The default `${{ secrets.GITHUB_TOKEN }}`
+won't cut it! You can [generate a PAT] in your GitHub Settings.
 
-Unfortunately, `${{ github.token }}` does not have access to updating wiki
-repositories, so it is necessary to create a new PAT.
+For example, if you created spenserblack/gigantic-mega-project-wiki to hold the
+wiki and you want to publish it to the GitHub wiki that belongs to _another
+repository_ like spenserblack/gigantic-mega-project, you'd use a step like this
```

Comment:
```suggestion
For example, if you created me/gigantic-mega-project-wiki to hold the
wiki and you want to publish it to the GitHub wiki that belongs to _another
repository_ like me/gigantic-mega-project, you'd use a step like this
```
Nitpick: I'd rather not use my own name as placeholders. Other options are `@mona` (frequently used placeholder in GH docs), `@ghost`, or `@octocat`.

Reply:
I like octocat, let's go with that :octocat:

Reply:
Fixed in https://github.com/jcbhmr/actions-wiki/commit/b4e7af66c4a87b0a9cfeefc466a00742138a2177